packages/eslint-config-turbo/src/index.ts

```diff
@@ -2,4 +2,5 @@ const config = {
   extends: ["plugin:turbo/recommended"],
 };
 
-module.exports = config;
+// eslint-disable-next-line import/no-default-export -- ESLint convention is a default export
+export default config;
```

Comment:
bunchee wasn't converting `module.exports` to `export default` for the ESM entrypoints. If I make it `export default`, it downlevels to CJS correctly.